tests/unit/storage_v2_durability_inmemory.cpp

```diff
@@ -2972,3 +3006,43 @@ TEST_P(DurabilityTest, ConstraintsRecoveryFunctionSetting) {
           &variant_existence_constraint_creation_func);
   MG_ASSERT(pval_existence, "Chose wrong type of function for recovery of existence constraint data");
 }
+
+// NOLINTNEXTLINE(hicpp-special-member-functions)
+TEST_P(DurabilityTest, EdgeTypeIndexRecovered) {
+  if (GetParam() == true) {
```

Comment:
Instead of placing all function body under one if, maybe we can reverse condition of if and make early return?